.circleci/config.yml

```diff
@@ -81,12 +81,25 @@ jobs:
         at: .
     - run: cp -v go/bin/random playbooks/files/random
     - run: cp -rv rules dashboards playbooks/files/
-    - run: sudo pip install --cache-dir ~/.cache/pip -q -r requirements.txt
+    - run: sudo pip install --cache-dir ~/.cache/pip -q -r molecule/test-requirements.txt
     - run: ansible-galaxy install -f -r roles/requirements.yml
     # Remove encrypted file so ansible won't try to decrypt it
     - run: rm group_vars/grafana/vault
-    - run: |
-        ansible-playbook --check -e "{'grafana_security': {'admin_user': 'admin', admin_password: 'secret' }}" site.yml
+    - setup_remote_docker
+    # - run: |
+    #    ansible-playbook -e "{'grafana_security': {'admin_user': 'admin', admin_password: 'secret' }}" site.yml
+    - run: molecule --debug dependency
+    - run: molecule --debug lint
+    - run: molecule --debug cleanup
+    - run: molecule --debug destroy
+    - run: molecule --debug syntax
+    - run: molecule --debug create
+    - run: molecule --debug prepare
+    - run: molecule --debug converge
+    # - run: molecule --debug idempotence
```

Comment:
Is this intended to work? Maybe put a TODO comment here?

Reply:
The existing playbooks don't have errors but seem to cause changes during the run, therefore causing idempotence to fail.   I believe it was related to the external security hardening roles rather than the prometheus roles.   

Added simply a "TODO" comment, to indicate it should be investigated further.  Probably not a show-stopper.

btw, please squash commit